src/ontology/mondo-edit.obo

```diff
@@ -1922,16 +1922,16 @@ subset: gard_rare {source="MONDO:GARD"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "GLUT1 deficiency syndrome" EXACT CLINGEN_LABEL []
-synonym: "GLUT1DS" EXACT ABBREVIATION [https://orcid.org/0000-0002-6601-2165]
+synonym: "GLUT1DS" EXACT ABBREVIATION [https://orcid.org/0000-0002-6601-2165, https://www.epilepsydiagnosis.org/aetiology/metabolic-groupoverview.html]
 xref: DOID:0070560 {source="MONDO:equivalentTo"}
 xref: MEDGEN:337833 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: NANDO:1200799 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0011-764X", source="https://orcid.org/0000-0002-0170-9172"}
 xref: OMIMPS:606777 {source="MONDO:equivalentTo"}
 xref: UMLS:C1847501 {source="MONDO:equivalentTo", source="MEDGEN:337833", source="MONDO:MEDGEN"}
 is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
-is_a: MONDO:0100033 {source="https://orcid.org/0000-0001-8486-0558"} ! metabolic epilepsy
+is_a: MONDO:0100033 {source="https://orcid.org/0000-0001-8486-0558", source="https://www.epilepsydiagnosis.org/aetiology/metabolic-groupoverview.html#glucose"} ! metabolic epilepsy
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0000188 {source="MONDO:CLINGEN"}
-relationship: disease_has_basis_in_disruption_of GO:0005355 ! obsolete glucose transmembrane transporter activity
+relationship: disease_has_basis_in_disruption_of GO:0055056 ! D-glucose transmembrane transporter activity
```

Comment:
Could you please add a source for this axiom?

Reply:
I don't know the source, there wasn't one there before. It probably came from Chris.